doc/source/drivers/vector/openfilegdb.rst

```diff
@@ -56,12 +62,102 @@ can be used as special SQL requests to get respectively the definition
 and metadata of a FileGDB table as XML content (only available in
 Geodatabases created with ArcGIS 10 or above)
 
+The "CREATE INDEX idx_name ON layer_name(field_name)" SQL request can be
+used to create an attribute index. idx_name must have 16 characters or less,
+start with a letter and contain only alpha-numeric characters or underscore.
+
+The "RECOMPUTE EXTENT ON layer_name" SQL request can be used to trigger
+an update of the layer extent in layer metadata. This is useful when updating
+or deleting features that modify the general layer extent.
+
+The "REPACK" or "REPACK layer_name" SQL requests can be used respectively to
+compact the whole database or a given layer. This is useful when doing editions
+(updates or feature deletions) that may leave holes in .gdbtable files. The REPACK
+command causes the .gdbtable to be rewritten without holes. Note that compaction
+does not involve extent recomputation.
+
 Dataset open options
 --------------------
 
 -  **LIST_ALL_TABLES**\ =YES/NO: This may be "YES" to force all tables,
    including system and internal tables (such as the GDB_* tables) to be listed (since GDAL 3.4)
 
+Dataset Creation Options
+------------------------
+
+None.
+
+Layer Creation Options
+----------------------
+
+-  **FEATURE_DATASET**: When this option is set, the new layer will be
+   created inside the named FeatureDataset folder. If the folder does
+   not already exist, it will be created.
+-  **LAYER_ALIAS**\ =string: Set layer name alias.
```

Comment:
```suggestion
-  **LAYER_ALIAS** =string: Set layer name alias.
```

Reply:
actually the best rendering is got with ``**LAYER_ALIAS**\=string``. The escape character is necessary after the end of the bold markup if there's not space, otherwise Sphinx complains.